### DISCUSSION TITLE
Can I get width and height of buffered images?

### DISCUSSION THREAD

**Discussion by Masynchin** [route/mogrify#107] [Q&A]
I want to render image blocks and then combine them into big one. To do this I need to calculate width and height of buffered images. I wrote [this code](https://github.com/Masynchin/ScheduleDrawer/blob/7d0ed28050365588c4870cb96e0ab5fc5549c319/lib/schedule_drawer.ex) with images manipulations that I looking for

**Comment by route** [comment]
Isn't it what you need to know in advance before combining them? `mogrify` gets info from the file, but since you don't open anything there's no data

**Reply by Masynchin** [reply]
> `mogrify` gets info from the file, but since you don't open anything there's no data

Doesn't the `create(buffer: true)` creates image with height and width? I think if there is buffer with some data, so it can have height and width
